OpenSim/Examples/ControllerExample/ControllerExample.cpp

```diff
@@ -317,8 +318,11 @@ int main()
         manager.integrate( si );
 
         // Save the simulation results.
-        osimModel.printControlStorage( "tugOfWar_controls.sto" );
-        manager.getStateStorage().print( "tugOfWar_states.sto" );
+        auto controlsTable = osimModel.getControlsTable();
+        MOTFileAdapter::write(controlsTable, "tugOfWar_controls.mot");
```

Comment:
I think it is confusing and unnecessary to permit the rename simply to revert in subsequent PR. Why not revert to `.mot` now and then rename `MOTFileAdpater` to `STOFileAdapter` since we are certain it handles `.sto` files. The correctness of the `MOTFileAdapter` to read SIMM generated motion files is of relatively low priority, so I don't want you to go down that route at present. I am OK with using our `STOFileAdapter` to read in our version of `.mot` files.